### PR TITLE
Added initial batchUpload method for ProjectService without category …

### PR DESCRIPTION
**Pull request by jacobj10**
I've already started on a batchupload endpoint, but thats just all parsing a file upload. My TS is pretty rusty so please let me know all the errors, category (or what I think should work) is commented out, lets go one step at a time...

I will be progressively adding to this pr, just want some feedback

### CODE REVIEW

File: src/server/controllers/ProjectService.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                //         `SELECT category_id FROM
+                //     categories WHERE name IN(:categories)`,
+                //         {
+                //             replacements: {
+                //                 projects: Array.from(project.categories),
+                //             },
+                //             type: sequelize.QueryTypes.SELECT,
+                //         });
+                // projectDetails.categories = projectCategoryInfo;
+                console.log(projectDetails);
```
Comment (AJLiu) [inline src/server/controllers/ProjectService.ts]: comment this out or use a debug-level log so this doesn't flood prod logging

**Comment by jacobj10** [comment]
@AJLiu Bump for updates. Still have to work on endpoint, but are the graphql endpoints you described in requested changes part of https://github.com/HackGT/ballot/issues/25 ?

**Comment by AJLiu** [comment]
re: #25 yeah those
And yeah it looks great, just need to do the last part of uploading the file!

**Comment by jacobj10** [comment]
Bump. Added the endpoint, all that is left should be graphql endpoints which should be ez

**Comment by AJLiu** [comment]
Will review tonight

**Pull request merged**